zypp/KeyRing.h

```diff
@@ -70,11 +70,13 @@ namespace zypp
       KEY_TRUST_AND_IMPORT
     };
 
+    const char * ACCEPT_PACKAGE_KEY_REQUEST = "KeyRingReport/AcceptPackageKey";
+
     /**
      * Ask user to trust and/or import the key to trusted keyring.
      * \see KeyTrust
      */
-    virtual KeyTrust askUserToAcceptKey( const PublicKey &key, const KeyContext &keycontext = KeyContext() );
+    virtual KeyTrust askUserToAcceptKey(const PublicKey &key, const KeyContext &keycontext = KeyContext() );
```

Comment:
You don't use 'git gui'to create the commits, do you? ('unstage lines' and Ctrl-J 'revert changes' makes it easy to omit unintended changes).

Reply:
Reverted